src/benchmarklib/benchmark_runner.cpp

```diff
@@ -400,10 +400,10 @@ cxxopts::Options BenchmarkRunner::get_basic_cli_options(const std::string& bench
   // some point. The way this is solved here is not really nice, but as the TPC-C benchmark binary has just a main
   // method and not a class, retrieving this default value properly would require some major refactoring of how
   // benchmarks interact with the BenchmarkRunner. At this moment, that does not seem to be worth the effort.
-  const auto default_mode = (benchmark_name == "TPC-C Benchmark" ? "Shuffled" : "Ordered");
+  const auto* const default_mode = (benchmark_name == "TPC-C Benchmark" ? "Shuffled" : "Ordered");
 
   // TPC-C does not support binary caching
-  const auto default_dont_cache_binary_tables = (benchmark_name == "TPC-C Benchmark" ? "true" : "false");
+  const auto* const default_dont_cache_binary_tables = (benchmark_name == "TPC-C Benchmark" ? "true" : "false");
```

Comment:
What? A const pointer to a const object?

Reply:
In the end, they both mean the same thing. `const auto` gets deduced to `const char* const`. I have no idea why clang-tidy suddenly wants to make this more explicit, but I think we can't turn it off without turning off other "this could be const" warnings too